src/TreeToTS/templates/javascript/functions.ts

```diff
@@ -4,7 +4,7 @@ export const javascriptFunctions = (env: Environment) => `
   export const ScalarResolver = (scalar, value) => {
     switch (scalar) {
       case 'String':
-        return  \`"\${value.replace(/"/g, '\\\\\\"')}"\`;
+        return  \`"\${(typeof value === 'string' ? value : \`\${value}\`).replace(/"/g, '\\\\\\"')}"\`;
```

Comment:
I think left and right of expression should be swapped can you check it ? so `c`?`A`:`B` should be c?`B`:`A` as if value is a string we should replace all `"` when its stringified json

Reply:
I tested and this works because the `.replace(/"/g, '\\"')` is performed on the result of
```typescript
(typeof value === 'string' ? value : `${value}`)
```
so regardless of whether `value` is a `string` or not, after it goes through this ternary expression, it will have all the occurrences of `"` replaced.

However, we can change the
```typescript
`${value}`
```
to `JSON.stringify(value)` instead if you'd feel it's more standard